runtime/layers/fpm-dev/Dockerfile

```diff
@@ -1,14 +1,28 @@
 ARG PHP_VERSION
-FROM bref/php-$PHP_VERSION-fpm
+FROM bref/build-php-$PHP_VERSION as build_extensions
 
+RUN pecl install xdebug
+RUN cp $(php -r "echo ini_get('extension_dir');")/xdebug.so /tmp
+
+FROM bref/php-${PHP_VERSION}-fpm as build_dev
+
+USER root
+COPY --from=build_extensions /tmp/*.so /tmp/
+RUN cp /tmp/*.so $(php -r "echo ini_get('extension_dir');")
+
+FROM bref/php-${PHP_VERSION}-fpm
+
+COPY --from=build_dev  /opt /opt
```

Comment:
Can we replace this:

```Dockerfile
FROM bref/php-${PHP_VERSION}-fpm as build_dev

USER root
COPY --from=build_extensions /tmp/*.so /tmp/
RUN cp /tmp/*.so $(php -r "echo ini_get('extension_dir');")

FROM bref/php-${PHP_VERSION}-fpm
```

with this:

```Dockerfile
FROM bref/php-${PHP_VERSION}-fpm

COPY --from=build_extensions /tmp/*.so /tmp/
RUN cp /tmp/*.so $(php -r "echo ini_get('extension_dir');")
```

?

That way it's simpler, it avoids going through another intermediary container. Or isthere a reason not to do that?

Reply:
Tried it first. But the lambci container is defined to use some other user which has no rights to write to the extension folder. This is the reason for first change the user to root then copy to extension folder

Reply:
OK noted, thanks!